enterprise/frontend/src/embedding-sdk/config.ts

```diff
@@ -1,5 +1,7 @@
 export const DEFAULT_FONT = "Lato";
 export const EMBEDDING_SDK_ROOT_ELEMENT_ID = "metabase-sdk-root";
+export const EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID =
```

Comment:
I added a new ID specific for the portal container for two reasons:
- it's technically not the root of the sdk anymore so it was a lie
- related to the above, the e2e tests were using that ID to scope down selectors, I decided not to change that so we won't have conflicts with the tests @WiNloSt has in his branch